spec/post_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'posts/index', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Developer',
+                        posts_counter: 0)
+    @first_post = Post.create(author: @user, title: 'Hello', text: 'This is my first post', comments_counter: 5,
+                              likes_counter: 0)
+    5.times do |i|
+      Comment.create(text: "This is comment ##{i}", user_id: @user.id, post_id: @first_post.id)
+    end
+    visit user_posts_path(@user.id, @first_post)
+  end
+
+  it 'displays post text' do
+    expect(page).to have_content('This is my first post')
+  end
+
+  it 'displays post title' do
+    expect(page).to have_content('Hello')
+  end
+
+  it 'displays first comment' do
+    expect(page).to have_content('This is comment #1')
```

Comment:
- [ ] In the test for `User post index page:`,  you did a good job, but you're missing two tests.
- "I can see some of the post's body."
- "When I click on a post, it redirects me to that post's show page."

### View:
![image](https://github.com/SFarida/rails_blog_application/assets/50990925/94b75c2e-5048-499c-8ddc-cb54440cabda)


- [ ] Kindly add these test to complete the good job you've done so far 👏

Reply:
Hi @Shedrack-Sunday  thank you for the remark, let me fix this.